projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -314,3 +334,28 @@ export function buildPrompt( {
 
 	return prompt;
 }
+
+type GetPromptOptionsProps = {
+	content: string;
+};
+
+/**
+ * Retunr a prompt based on the type and options
+ *
+ * @param {PromptTypeProp} type           - The type of prompt.
+ * @param {GetPromptOptionsProps} options - The prompt options.
+ * @returns {Array< PromptItemProps >}      The prompt.
+ */
+export function getPrompt(
+	type: PromptTypeProp,
+	options: GetPromptOptionsProps
+): Array< PromptItemProps > {
+	return buildPrompt( {
+		type,
+		generatedContent: options.content,
+		postContentAbove: options.content,
```

Comment:
Maybe we need two different option properties for "content" and "content above". I see this being the same for some prompts, but if we plan to use `getPrompt` everywhere, we will need both options, right?